Sources/XcbeautifyLib/CaptureGroups.swift

```diff
@@ -1866,3 +1866,368 @@ struct SwiftDriverJobDiscoveryCompilingCaptureGroup: CaptureGroup {
         self.project = project
     }
 }
+
+struct SwiftTestingRunStartedCaptureGroup: CaptureGroup {
+    static let outputType: OutputType = .result
+
+    /// Regular expression to capture the start of a test run.
+    /// $1 = symbol
+    /// $2 = message
+    static let regex = Regex(pattern: #"^(􀟈)\s*(Test run started\.)$"#)
+
+    let symbol: String
+    let message: String
+
+    init?(groups: [String]) {
+        guard let symbol = groups[safe: 0],
+              let message = groups[safe: 1] else { return nil }
+        self.symbol = symbol
+        self.message = message
+    }
+}
+
+struct SwiftTestingRunCompletionCaptureGroup: CaptureGroup {
+    static let outputType: OutputType = .result
+
+    /// Regular expression to capture the number of tests and total time.
+    /// $1 = symbol
+    /// $2 = number of tests
+    /// $3 = total time in seconds
+    static let regex = Regex(pattern: #"^(􁁛)\s*Test run with (\d+) tests passed after ([\d.]+) seconds\.$"#)
+
+    let symbol: String
+    let numberOfTests: String
+    let totalTime: String
+
+    init?(groups: [String]) {
+        guard let symbol = groups[safe: 0],
+              let numberOfTests = groups[safe: 1],
+              let totalTime = groups[safe: 2] else { return nil }
+        self.symbol = symbol
+        self.numberOfTests = numberOfTests
+        self.totalTime = totalTime
+    }
+}
+
+struct SwiftTestingRunFailedCaptureGroup: CaptureGroup {
+    static let outputType: OutputType = .result
+
+    /// Regular expression to capture the number of tests, total time, and the number of issues.
+    /// $1 = symbol
+    /// $2 = number of tests
+    /// $3 = total time in seconds
+    /// $4 = number of issues
+    static let regex = Regex(pattern: #"^(􀢄)\s*Test run with (\d+) tests failed after ([\d.]+) seconds with (\d+) issue[s]?\.$"#)
+
+    let symbol: String
+    let numberOfTests: String
+    let totalTime: String
+    let numberOfIssues: String
```

Comment:
Can these instead be Ints/Double?

Reply:
I left `totalTime ` as a `String` here. It uses the colored extension on `String`. It uses the extension on `String` to do color for time. We could have the call a helper ext on `Double` or just cast back to `String` before using. Since there is no operation occurring on time not sure it's needed. Thoughts?